cmd/renterd/main.go

```diff
@@ -115,13 +122,13 @@ func main() {
 		autopilotCfg.busPassword = apiPassword
 	}
 	if workerCfg.enabled {
-		w, cleanup, err := newWorker(workerCfg, getWalletKey())
+		w, cleanup, err := newWorker(workerCfg, walletKey)
```

Comment:
The worker uses the `walletKey` to derive a `workerKey` but we're currently not using that for anything...
I'm not sure what the worker would need a key for so maybe we can remove it for the time being. Might be that Luke had plans for it though.